lib/Dialect/Triton/IR/Ops.cpp

```diff
@@ -543,6 +543,8 @@ OpFoldResult SplatOp::fold(FoldAdaptor adaptor) {
   auto value = adaptor.getSrc();
   if (!value)
     return {};
+  if (!isa<FloatAttr, IntegerAttr>(value))
+    return {};
```

Comment:
Can you include why you need this change in our PR description?

Reply:
Added a short explanation and the culprit llvm commit that introduced these changes, PTAL.

Reply:
makes sense, thanks